src/components/Launch/LaunchWorkflowForm/inputHelpers/utils.ts

```diff
@@ -14,3 +15,13 @@ export function extractLiteralWithCheck<T>(
     }
     return value as T;
 }
+
+/** Converts a value within a collection to the appropriate string
+ * representation. Some values require additional quotes.
+ */
+export function collectionChildToString(type: InputType, value: any) {
```

Comment:
Could be worthwhile to add an explicit return type of `string` here so that future code changes do not accidentally return unexpected types

Reply:
That's a good idea. Code consuming this expects a string, so changes here would still result in a type error. But I can't guarantee that any new usage of this function would also have such a strict type expectation.